map/track.hpp

```diff
@@ -55,6 +55,7 @@ class Track : public df::UserLineMark
   size_t GetLayerCount() const override;
   dp::Color GetColor(size_t layerIndex) const override;
   void SetColor(dp::Color color);
+  void SetColor(uint32_t color);
```

Comment:
Do we need this function if dp::Color is constructible from uint32_t rgba?

Reply:
Oh yes, sorry i missed that, thanks will modify it